WORKSPACE

```diff
@@ -126,6 +126,7 @@ vaticle_typedb_artifact()
 vaticle_typedb_cluster_artifact()
 
 # Load maven
+load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_artifacts = "artifacts")
```

Comment:
we need to load `common` maven requirements as well

Reply:
why?

Reply:
because we depend on all of common, and common now has a sub-library (without its own package) that requires maven dependencies!